index.html

```diff
@@ -718,24 +736,26 @@ <h3>Resource Timing Attributes</h3>
 <h2>Creating a resource timing entry</h2>
 <p>To <dfn export="">mark resource timing</dfn> given a
 [=/fetch timing info=] |timingInfo|, a DOMString
-|requestedURL|, a DOMString |initiatorType| and a <a>global object</a> |global|:
+|requestedURL|, a DOMString |initiatorType| a <a>global object</a> |global|, and a string |cacheMode|,
```

Comment:
s/string/DOMString/ ?

Reply:
Also, are you planning to change the callers?

Reply:
Already changed FETCH (Waiting for review): https://github.com/whatwg/fetch/pull/1202. Will modify  HTML for nav-timing afterwards.